### PR TITLE
Add configuration for editors

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+[*]
+end_of_line = lf
+charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[{*.js,*.json,*.yml}]
+indent_size = 2
+indent_style = space
```
Comment (lundibundi) [inline .editorconfig:12]: Why not put these 2 in just `*`?

Comment (tshemsedinov) [reply .editorconfig:12]: I believe we will not add other languages except ts typings in most of our packages so * is very general mask. Let's add `*.ts` and maybe `*.h,*.cc,*.c`

Comment (tshemsedinov) [reply .editorconfig:12]: I'd like to have common `.editorconfig` for almost all modules in Metarhia

**Comment by tshemsedinov** [comment]
Landed in https://github.com/metarhia/web-locks/commit/3ebd9187bde8f972c50c8687bf2a4bfd136578d0

**Pull request closed without merging**